website/docs/d/alert_configurations.html.markdown

```diff
@@ -64,11 +64,124 @@ In addition to all arguments above, the following attributes are exported:
+
+* `metric_name` - Name of the metric to check. The full list being quite large, please refer to atlas docs [here for general metrics](https://docs.atlas.mongodb.com/reference/alert-host-metrics/#measurement-types) and [here for serverless metrics](https://www.mongodb.com/docs/atlas/reference/api/alert-configurations-create-config/#serverless-measurements)
+
+* `operator` - Operator to apply when checking the current metric value against the threshold value.
+  Accepted values are:
+    - `GREATER_THAN`
+    - `LESS_THAN`
+
+* `threshold` - Threshold value outside of which an alert will be triggered.
+* `units` - The units for the threshold value. Depends on the type of metric.
+  Refer to the [MongoDB API Alert Configuration documentation](https://www.mongodb.com/docs/atlas/reference/api/alert-configurations-get-config/#request-body-parameters) for a list of accepted values.
+* `mode` - This must be set to AVERAGE. Atlas computes the current metric value as an average.
+
+### Threshold Config
+* `operator` - Operator to apply when checking the current metric value against the threshold value.
+  Accepted values are:
+    - `GREATER_THAN`
+    - `LESS_THAN`
+
+* `threshold` - Threshold value outside of which an alert will be triggered.
+* `units` - The units for the threshold value. Depends on the type of metric.
+  Refer to the [MongoDB API Alert Configuration documentation](https://www.mongodb.com/docs/atlas/reference/api/alert-configurations-get-config/#request-body-parameters) for a list of accepted values.
+
+### Notifications
+Notifications to send when an alert condition is detected.
+
+* `api_token` - Slack API token. Required for the SLACK notifications type. If the token later becomes invalid, Atlas sends an email to the project owner and eventually removes the token.
+* `channel_name` - Slack channel name. Required for the SLACK notifications type.
+* `datadog_api_key` - Datadog API Key. Found in the Datadog dashboard. Required for the DATADOG notifications type.
+* `datadog_region` - Region that indicates which API URL to use. Accepted regions are: `US`, `EU`. The default Datadog region is US.
+* `delay_min` - Number of minutes to wait after an alert condition is detected before sending out the first notification.
+* `email_address` - Email address to which alert notifications are sent. Required for the EMAIL notifications type.
+* `email_enabled` - Flag indicating email notifications should be sent. Atlas returns this value if `type_name` is set  to `ORG`, `GROUP`, or `USER`.
+* `flowdock_api_token` - The Flowdock personal API token. Required for the `FLOWDOCK` notifications type. If the token later becomes invalid, Atlas sends an email to the project owner and eventually removes the token.
+* `flow_name` - Flowdock flow name in lower-case letters. Required for the `FLOWDOCK` notifications type
```

Comment:
```suggestion
* `flow_name` - Flowdock flow name in lower-case letters. Required for the `FLOWDOCK` notifications type.
```

Reply:
question here: is flowdock EOL? wondering if we should include this